inc/Subscriptions/BulkSubscriptionSender.php

```diff
@@ -6,7 +6,6 @@
 
 use dokuwiki\ChangeLog\PageChangeLog;
 use dokuwiki\Input\Input;
-use DokuWiki_Auth_Plugin;
 
```

Comment:
still used at line 32 in phpdocs, I guess rector does not recognize yet these in phpdocs?

Reply:
Probably not. But it seems the `use` is not needed for Idea to understand the type hint